assets/src/edit-story/elements/image/output.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import StoryPropTypes from '../../types';
+import { PAGE_WIDTH, PAGE_HEIGHT } from '../../constants';
+import { editorPixels } from '../../units';
+import { getImgProps } from './util';
+
+/**
+ * Returns AMP HTML for saving into post content for displaying in the FE.
+ */
+function ImageOutput( {
+	element: { src, origRatio, scale, focalX, focalY },
+	box: { width: vw, height: vh },
+} ) {
+	// Width and height I taken from the basis of 100% taking into account the
```

Comment:
Nit: should this be comment be `... and height are taken ... ` instead of `"... I taken ... "`?

Reply:
Definitely!